src/Symfony/Component/PropertyAccess/Tests/StringUtilTest.php

```diff
@@ -21,7 +21,9 @@ public function singularifyProvider()
         // see http://www.scribd.com/doc/3271143/List-of-100-Irregular-Plural-Nouns-in-English
         return array(
             array('accesses', 'access'),
+            array('access', 'access'),
```

Comment:
Yes thats right all new tests are singular words. But because of
```
        // tags (tag)
        array('s', 1, true, true, ''),
```
in pluralmap this words are handled as plural and the last "s" will be removed